readme.txt

```diff
@@ -72,6 +72,18 @@ Yes, you can track your MapKit JS useage on the [MapKit JS Developer Dashboard](
 
 == Changelog ==
```

Comment:
The changelog is getting a little large, let's keep the last two or three releases in the wp.org readme and refer people to the GitHub changelog for the full log.

Linking to a changelog is suggested by [the plugin readme docs](https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/#file-size), so won't fall fowl of the plugin guidelines.

Reply:
Updated 2549d61